builder-base/scripts/install_notation.sh

```diff
@@ -40,4 +42,18 @@ function install_notation() {
     time upx --best --no-lzma $USR_BIN/notation
 }
 
-[ ${SKIP_INSTALL:-false} != false ] || install_notation
+function install_aws_signer_plugin() {
+    wget \
+        --progress dot:giga \
+        $AWS_SIGNER_PLUGIN_URL
+    tar -xzvf $AWS_SIGNER_PLUGIN_NAME
+    mv notation-com.amazonaws.signer.notation.plugin com.amazonaws.signer.notation.plugin
+    mkdir $HOME/.config
+    mkdir $HOME/.config/notation
+    mkdir $HOME/.config/notation/plugins
+    mkdir $HOME/.config/notation/plugins/com.amazonaws.signer.notation.plugin
```

Comment:
Instead of all these lines
```suggestion
    mkdir -p $HOME/.config/notation/plugins/com.amazonaws.signer.notation.plugin
```